pkg/controller/plan/migration.go

```diff
@@ -1522,16 +1527,13 @@ func (r *Migration) setDataVolumeCheckpoints(vm *plan.VMStatus) (err error) {
 			return
 		}
 	}
-	var vmCr VirtualMachine
-	found := false
-	if vmCr, found = r.vmMap[vm.ID]; !found {
```

Comment:
so now that `r.vmMap` is (rightly) not used we can drop the first few lines above, right?

Reply:
right